### PR TITLE
Trrack vis

### PR DESCRIPTION
**Pull request by JakeWags**
### Give a longer description of what this PR addresses and why it's needed
Adds Trrack Vis and provenance graph view as well as changes the icons on the element sidebar.

### Are there any additional TODOs before this PR is ready to go?

TODOs:
- [ ] ...

### CODE REVIEW

File: packages/upset/src/components/ElementView/ElementSidebar.tsx
Comment (JackWilb) [inline packages/upset/src/components/ElementView/ElementSidebar.tsx:107]: Why add 5 here? Can this not just be included in the `yOffset` variable?

File: packages/upset/src/components/ElementView/ElementSidebar.tsx
Comment (JackWilb) [inline packages/upset/src/components/ElementView/ElementSidebar.tsx:143]:
```suggestion
          </IconButton>
          : <IconButton
```

**Review by JackWilb — COMMENTED** [review]
A couple of question. Waiting on some pictures for approval

File: packages/upset/src/components/ElementView/ElementSidebar.tsx
Comment (JakeWags) [reply packages/upset/src/components/ElementView/ElementSidebar.tsx]: Addressed in commit 152aa75

**Comment by JakeWags** [comment]
# Element sidebar
## Sidebar
image

## Fullscreen
image

# Trrack Vis
image

**Review by JackWilb — APPROVED** [review]

**Pull request merged**
